Learninglocker (#236)
openfun/arnold: CHANGELOG.md

### Added

- Create ConfigMaps to be consumed in environment variables in a DeploymentConfig.

> - Add learning locker in the list of configured application.

Reviewer: ```suggestion
- Add LearningLocker to the list of supported applications.
```

> - Activate and configure learning locker for eugene customer in development environment.

Reviewer: ```suggestion
- Activate and configure LearningLocker for the eugene customer in development & CI environments.
```
